go/tools/builders/nogo_main.go

```diff
@@ -234,7 +234,7 @@ func checkPackage(analyzers []*analysis.Analyzer, packagePath string, packageFil
 
 	// Process diagnostics and encode facts for importers of this package.
 	diagnostics := checkAnalysisResults(roots, pkg)
-	facts := pkg.facts.Encode()
+	facts := pkg.facts.Encode(true)
```

Comment:
```suggestion
	facts := pkg.facts.Encode(true /* skipMethodSorting */)
```
Could we either attach a meaning to this? I don't want future confusion from API breaking changes to this call.

Naked parameters in function calls can hurt readability.

Reply:
Added